app/controllers/api/applications_controller.rb

```diff
@@ -42,8 +42,8 @@ def find_service
     @service = accessible_services.find params[:service_id]
   end
 
-  def accessible_plans
-    super.where(issuer: @service)
+  def find_plans
```

Comment:
Overriding the `find_plans` in order to avoid adding `.stock`.
As I understand the logic, the idea is that if there is just one single plan (I guess that's not commonly used, but still), there is no need to show the **Plan** column, because well, it will always be the same.

But for that purpose I think it is more correct to check all plans (including the custom ones), and not just "stock" plans (non-custom).